gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -459,6 +459,9 @@
       "source.querybased.promoteUnsignedIntToBigInt";
   public static final boolean DEFAULT_SOURCE_QUERYBASED_PROMOTE_UNSIGNED_INT_TO_BIGINT = false;
 
+  public static final String ENABLE_DELIMITED_IDENTIFIER = "enable.delimited.identifier";
+  public static final boolean DEFAULT_DELIMITED_IDENTIFIER = false;
```

Comment:
Convention is DEFAULT_<key>, so this should be DEFAULT_ENABLE_DELIMITED_IDENTIFIER.